docs/administration-guide/data-permissions.md

```diff
@@ -28,50 +28,59 @@ You can click on any cell in the permissions table to change a group’s access
 
 ### Granular access
 
-**Granular access** allows administrators to explicitly set access to tables or schemas within a database. In practice, this means that:
+**Granular access** allows administrators to explicitly set data access to tables or schemas within a database, with "data access" here meaning the ability to create questions using the graphical query builder. In practice, this means that:
 
-- Admins can set the groups access to individual tables to either **Unrestricted**, **No self-service**, or **Sandboxed** access.
+- Admins can set the group's access to individual tables to either **Unrestricted**, **No self-service**, or **Sandboxed** access.
 - If a new table gets added to this database in the future, the group won't get access to that new table. An administrator would need to explicitly grant access to that table.
 
+Note that [Block](#block-access) access is unavailable for individual tables/schemas. Block is a database-level setting; you can only block the entire database.
+
 ### No self-service access
 
-**No self-service** prevents people in a group from creating new ad hoc queries or questions based on this data, or from seeing this data in the Browse Data screen. Groups with this level of access can still see saved questions and charts based on this data in Collections they have access to.
+**No self-service** prevents people in a group from using the graphical query builder to create new questions that query that database, or from seeing this database in the Browse Data section of your Metabase. Groups with No self-service access can still see saved questions that query this data if they 1) have access to the appropriate collection, and 2) aren't in a group with [blocked access](#block-access) to the database.
 
 ### Block access
 
 {% include plans-blockquote.html feature="Block access" %}
 
-**Block** ensures people can’t ever see the data from this database, regardless of their permissions at the Collection level. So if they want to see a question in a collection that have access to, but that question uses data from a database that's been blocked for that person's group, then they won't be able to see that question.
+**Block** ensures people in a group can’t see the data from this database, regardless of their permissions at the collection level. 
+
+Even if a question is in a collection that the group has access to, but that question queries a database that is blocked for that group, people in that group won't be able to view that question _unless_ they're in another group with the relevant data permissions. Essentially, what Block does is make collections permissions insufficient to view a question.
```

Comment:
> Essentially, what Block does is make collections permissions insufficient to view a question.

@jeff-bruemmer do you think some people might be confused by this and expect that they will either 1) not see a question in a collection or 2) not be able to open it?

The crucial part here is that they are not able to see the data in that question.
They can still see question's name and description (and they can even change those if they have Curate permission level on a collection).